common/db/migrations/094_add_description.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'utils'
+
+Sequel.migration do
+
+  up do
+    alter_table(:repository) do
+     TextField :description
```

Comment:
Should this be add_column(:description, TextField)?

Reply:
I followed a precedent:

https://github.com/archivesspace/archivesspace/blob/master/common/db/migrations/044_add_subtitle.rb

Definitely works that way. Docs here show `$type` `$fieldname` in block as valid usage:

http://sequel.jeremyevans.net/rdoc/files/doc/migration_rdoc.html#label-A+Basic+Migration